deps/rabbitmq_management/priv/www/js/oidc-oauth/helper.js

```diff
@@ -4,18 +4,22 @@ var _management_logger;
 
 function oauth_initialize_if_required() {
     rabbit_port = window.location.port ? ":" +  window.location.port : ""
-    rabbit_base_uri = window.location.protocol + "//" + window.location.hostname + rabbit_port
+    rabbit_path_prefix = window.location.pathname.replace(/(\/js\/oidc-oauth\/.*$|\/+$)/, "")
```

Comment:
@CAPCHIK For now the path I have chosen is to document it. I wanted to keep the oidc callback onto separate html pages rather than having to handle them as sammy routes, i.e. with `#/js/...`. 

Once again thanks for reviewing and excellent feedback !